crypto/fipsmodule/ec/p224-64.c

```diff
@@ -180,31 +175,20 @@ static const p224_felem g_p224_pre_comp[2][16][3] = {
       {0x32477c61b6e8c6, 0xb46a97570f018b, 0x91176d0a7e95d1, 0x3df90fbc4c7d0e},
       {1, 0, 0, 0}}}};
 
-static uint64_t p224_load_u64(const uint8_t in[8]) {
-  uint64_t ret;
-  OPENSSL_memcpy(&ret, in, sizeof(ret));
-  return ret;
-}
 
 // Helper functions to convert field elements to/from internal representation
-static void p224_bin28_to_felem(p224_felem out, const uint8_t in[28]) {
-  out[0] = p224_load_u64(in) & 0x00ffffffffffffff;
-  out[1] = p224_load_u64(in + 7) & 0x00ffffffffffffff;
-  out[2] = p224_load_u64(in + 14) & 0x00ffffffffffffff;
-  out[3] = p224_load_u64(in + 20) >> 8;
-}
-
-static void p224_felem_to_bin28(uint8_t out[28], const p224_felem in) {
-  for (size_t i = 0; i < 7; ++i) {
-    out[i] = in[0] >> (8 * i);
-    out[i + 7] = in[1] >> (8 * i);
-    out[i + 14] = in[2] >> (8 * i);
-    out[i + 21] = in[3] >> (8 * i);
-  }
-}
 
+#if defined(AWSLC_FIPS)
+// Related to delocate issue with gcc-11
+__attribute__((noinline))
```

Comment:
Is this attribute portable enough? It's a GNU thing. I don't recall if we need to guard against other compilers.

Reply:
This file is not compiled on Windows:
* [BORINGSSL_HAS_UINT128](https://github.com/aws/aws-lc/blob/6a1e61a81bb5a1f77bcbf94a1b451b89d939d59b/crypto/internal.h#L160) is not defined for Windows:
```
#if (!defined(_MSC_VER) || defined(__clang__)) && defined(OPENSSL_64_BIT)
#define BORINGSSL_HAS_UINT128
```
* [p224-64.c](https://github.com/aws/aws-lc/blob/6a1e61a81bb5a1f77bcbf94a1b451b89d939d59b/crypto/fipsmodule/ec/p224-64.c#L34) requires `BORINGSSL_HAS_UINT128` to be set.
```
#if defined(BORINGSSL_HAS_UINT128) && !defined(OPENSSL_SMALL)
```